src/js/Keyboard.js

```diff
@@ -40,20 +40,11 @@ class Keyboard extends Component {
 
   onChange = input => {
     // Changes from button presses
-    let position = this.keyboard.caretPosition;
     var tabPattern = /\t/g;
-    var tabCount = ((input || '').match(tabPattern) || []).length * 8;
+    var tabCount = ((input || '').match(tabPattern) || []).length * 7;
```

Comment:
Since I am counting the tab character in the total tab character count, i need to add 7 more to the count (tab width being 8 spaces). When it was `length * 8` i was double counting some characters